aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt

```diff
@@ -0,0 +1,291 @@
+        val input = UNORDERED
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun quoteAttributes() {
+        val input = QUOTE
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun linkAttributes() {
+        val input = LINK
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun unknownAttributes() {
+        val input = UNKNOWN
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun listAttributes() {
+        val input = LIST
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun subAttributes() {
+        val input = SUB
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun supAttributes() {
+        val input = SUP
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun fontAttributes() {
+        val input = FONT
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun typefaceAttributes() {
+        val input = TT
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun bigAttributes() {
+        val input = BIG
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun smallAttributes() {
+        val input = SMALL
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun paragraphAttributes() {
+        val input = P
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun mixedAttributes() {
+        val input = MIXED + NESTED
+        val expected = MIXED + NESTED_REVERSED
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(expected, output)
+    }
+
+    //TODO: After fixing list item attribute preservation, uncomment this
```

Comment:
Yeah, it's that one. When we were merging the attributes PR with @khaykov's work on lists there was some issues with list items so we'll need to go back to it.